main.tf

```diff
@@ -33,6 +33,24 @@ resource "aws_s3_bucket" "postman_bucket_logs" {
   bucket = "${var.app_name}-postman-tests-${data.aws_caller_identity.current.account_id}-logs"
   acl    = "log-delivery-write"
   tags   = var.tags
+
+  lifecycle_rule {
+    id                                     = "AutoAbortFailedMultipartUpload"
+    enabled                                = true
+    abort_incomplete_multipart_upload_days = 10
+
+    expiration {
+      days                         = 0
```

Comment:
what does this do?

Reply:
It's a policy that Divvy will add if we don't. I don't know exactly what it does though. All our other modules have it on buckets.